includes/Plugins.php

```diff
@@ -341,6 +341,16 @@ final class Plugins {
 					'activate' => true,
 					'priority' => 300,
 				),
+				array(
+					'slug'     => 'jetpack',
+					'activate' => true,
+					'priority' => 250,
+				),
+				array(
+					'slug'     => 'wordpress-seo',
+					'activate' => true,
+					'priority' => 260,
+				),
```

Comment:
Aren't `jetpack` and `wordpress-seo` be part of the default list for all flows?

Reply:
Sure, lemme check that out and revert, Thanks.